src/api/config/routes.rb

```diff
@@ -16,8 +16,15 @@ def self.matches?(request)
 class APIMatcher
   def self.matches?(request)
     format = request.format.to_sym || :xml
-    # Rails.logger.debug "MATCHES #{format}"
-    format == :xml || format == :json
+
+    format == :xml || format == :json || APIMatcher.is_public_or_about(request)
+  end
+
+  def self.is_public_or_about(request)
+    if /\/public||\/about/ =~ request.fullpath
+      request.format = :xml
```

Comment:
I don't like that a supposedly boolean method have side effects... move the assignment to another place.